### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by KillNagaThul** [CyanideX/Unity-Dark#114]
**Suggestion**
Hello, Please add "Botany Pots"  (https://legacy.curseforge.com/minecraft/mc-mods/botany-pots) and "Botany Pots Tiers" (https://legacy.curseforge.com/minecraft/mc-mods/botany-pots-tiers)

**Comment by CyanideX** [comment]
https://github.com/Unity-Resource-Pack/Unity-Dark/issues/112#issuecomment-1636976240

**Comment by KillNagaThul** [comment]
![hopper_botany_pot_gui](https://github.com/Unity-Resource-Pack/Unity-Dark/assets/76072743/a33e4532-983b-49fb-9733-d8877be4f697)
![botany_pot_gui](https://github.com/Unity-Resource-Pack/Unity-Dark/assets/76072743/2167768b-45e8-44cc-ab77-5fab41fd58cf)

**Comment by CyanideX** [comment]
Are you submitting these? If so, please submit them in a PR.

**Comment by KillNagaThul** [comment]
yes I'm sending them to you, but I don't know where, I'm a graphic designer and I don't really understand how Github works
